examples/basic-dapp/index.js

```diff
@@ -3,6 +3,8 @@
 // https://github.com/Conflux-Chain/js-conflux-sdk#readme
 // eslint-disable-next-line import/no-unresolved
 // import {Conflux} from 'https://cdn.skypack.dev/js-conflux-sdk'
+import {iface} from '../../packages/browser-extension/build/popup/dist/@fluent-wallet/contract-abis/777.js'
+import {decode} from '../../packages/browser-extension/build/popup/dist/@fluent-wallet/base32-address/index.js'
```

Comment:
decouple this example with wallet

1. this demo is served as an dapp example for dapp developer
2. the build dir is empty without running yarn prod
3. this will broken the online version of this demo https://dapp-demo.fluentwallet.dev/

check out below links as example

https://github.com/Conflux-Chain/conflux-portal/blob/51a37674fc88f6d0b7326594e4c943e8984342f0/test/e2e/contract-test/contract.js#L1
https://github.com/metamask/test-dapp/tree/main

Reply:
what is the jssdk instance in fluent about `ConfluxJS` in portal